generated/.tailcallrc.graphql

```diff
@@ -368,6 +368,11 @@ directive @upstream(
   """
   httpCache: Int
   """
+  Providing httpCachePercentage allocates a maximum of that percentage of RAM to the 
+  HTTP cache. Defaults to `0` if unspecified.
+  """
+  httpCachePercentage: String
```

Comment:
```suggestion
  httpCachePercentage: Float
```

Reply:
Wont this break the current system as httpcache is u64?

Reply:
@tusharmath another reason I chose **_string_** was that , **_float_** does not support `Eq` And the _**upstream**_ data structure was designed to support `Eq` hence the workaround with string

Reply:
We can make breaking changes until we reach version 1

Reply:
> @tusharmath another reason I chose **_string_** was that , **_float_** does not support `Eq` And the _**upstream**_ data structure was designed to support `Eq` hence the workaround with string

@tusharmath I tried solving this issue but could not find a way the error is as follows  
![image](https://github.com/tailcallhq/tailcall/assets/81297089/113918b4-7f77-422d-a996-2bd850bbce92)

One thing I found was ordered float https://docs.rs/ordered-float/latest/ordered_float/ but this did not work as well, because it violated  _**schemars::JsonSchema**_ and **_Serialize_**

Reply:
@tusharmath all other comments addressed and changed as per suggestions only this has been taken in as a string as f32 is not possible

Reply:
Keep it as a `u64` with a range in between `0-100`. Anything beyond that will be considered as `100`.

Reply:
Ok